### PR TITLE
Various food fixes - Add seed items for lentils and beans

### PR DESCRIPTION
**Pull request by anothersimulacrum**
#### Summary
```SUMMARY: Bugfixes "Add seed items for lentils and raw beans"```
Also made condensed milk autolearn, added a recipe for dehydrating lentils, and formatted `GUIDE_COMESTIBLES.md`.

#### Purpose of change
Having a separate seed item for plants prevents exploits, such as plants lasting forever, and also prevents the player from accidentally using up all of a plant's seed that they were going to plant.

Condensed milk is a fairly easy recipe, and should be auto-learned.

Lentils didn't have a dehydrating recipes, and could only be dehydrated through the smoking rack previously.

`GUIDE_COMESTIBLES.md` appeared as one paragraph, but was formatted like it was a list. The headers were added because those lines were separated previously, but can be removed if necessary.

#### Describe the solution
JSON editing

### CODE REVIEW

File: data/json/items/comestibles/raw_veggy.json [hunk 1]
```diff
@@ -626,5 +626,47 @@
...
+    "name": "raw lentils",
+    "name_plural": "raw lentils",
+    "color": "light_green",
+    "spoils_in": "2 days",
+    "comestible_type": "FOOD",
+    "weight": 194,
+    "symbol": "%",
+    "healthy": -2,
+    "calories": 42,
+    "description": "Raw, uncooked lentils. They could be cooked.",
```
Comment (AMurkin) [inline data/json/items/comestibles/raw_veggy.json]:
```suggestion
    "description": "Raw, uncooked lentils.  They could be cooked.",
```

File: data/json/items/comestibles/raw_veggy.json [hunk 2]
```diff
@@ -626,5 +626,47 @@
...
     "volume": 1,
     "price": 0
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "raw_beans",
+    "name": "raw beans",
+    "name_plural": "raw beans",
+    "color": "brown",
+    "spoils_in": "2 days",
```
Comment (Night-Pryanik) [inline data/json/items/comestibles/raw_veggy.json]: Only two days? I thought they could last for much longer.

Comment (anothersimulacrum) [reply data/json/items/comestibles/raw_veggy.json]: This is based on the shelf life for [green beans](https://www.eatbydate.com/proteins/beans-peas/beans-shelf-life-expiration-date/). Dried beans, which exist in game, last for much longer, and are probably what you are thinking of.

Comment (Night-Pryanik) [reply data/json/items/comestibles/raw_veggy.json]: Oh, I got it.

**Pull request merged**